src/styles.css

```diff
@@ -42,13 +42,15 @@
 
 @media only screen and (min-width: 481px) {
   .login-card {
+    margin-top: 1rem;
     padding: 3rem;
     width: 26.375rem;
   }
 }
 
 @media only screen and (max-width: 480px) {
   .login-card {
+    margin-top: 1rem;
```

Comment:
Sure that looks good on mobile?

I would rather go with `margin-top: 2.5%` here to have the same as `width: 95%`.

The other thing we can do is to have `max-width: 100%` on the `svg[role="img"]` used on top. Otherwise, it looks strange on mobile.

Reply:
It looked better in mobile view as well.

![image](https://user-images.githubusercontent.com/38713281/81921846-dc611780-95f8-11ea-852d-e5fa5a4fa934.png)

Reply:
I will try the `margin-top: 2.5%` option as well

Reply:
The `margin-top: 2.5%` option looked better, Updated the PR.

![image](https://user-images.githubusercontent.com/38713281/81922300-8b055800-95f9-11ea-9465-81ca37516c54.png)

Reply:
@FlorianRappl The first one is addressed.
The second item - to have `max-width: 100%` on the `svg[role="img"]` used on top

max-width is already set for logo. 

```
.logo-img {
  margin-bottom: 3rem;
  max-width: 100%;
}
```

Reply:
But this class is not set.

![image](https://user-images.githubusercontent.com/1766191/81934795-d581d700-95ef-11ea-825d-990a9995f390.png)